crates/biome_js_syntax/src/jsx_ext.rs

```diff
@@ -408,6 +408,15 @@ impl AnyJsxElement {
         }
     }
 
+    pub fn get_attribute_inner_string_text(&self, name_to_lookup: &str) -> Option<String> {
```

Comment:
In the extension crate, _we try_ to document each function, and add doc tests too. Here's an example:

https://github.com/biomejs/biome/blob/8fd98d0e6dd794f3f5bfa4b6c264344b7b9a727a/crates/biome_js_syntax/src/identifier_ext.rs#L65-L76